### PR TITLE
Menu bar for mobile version

### PR DESCRIPTION
**Pull request by bahoz-ali**
**I worked hard to implement the following features for the mobile version:-**

1. When the user clicks the hamburger button, the mobile menu appears.
2. When the user clicks the close (X) button, the mobile menu disappears.
3. When the user clicks any of the mobile menu options, the mobile menu disappears.
4. When the user clicks any of the mobile menu options, a correct part of the page is displayed.

NOTE: **I have tried all possible ways to solve the linter error on GitHub actions but it's not working, however locally all the linters are passing. If you can assist in solving or give any idea it would be great. I asked the senior code reviewer and he told me to submit it for review. kindly review my Project. Thanks
**

**senior code reviewer told us to write this NOTE here.**

### CODE REVIEW

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Invalid Code Review Request ❌

Hi @bahoz-ali, This is @sja-thedude your code reviewer.

While you made a great effort in this project, I cannot proceed to review your code.

You have submitted a project, that doesn't pass all linters. Make sure all have a green checkmark.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: style.css [hunk 1]
```diff
@@ -53,20 +53,62 @@ a {
...
+  padding: 100px 0 0 30px;
+  margin: 0;
 }
 
 .nav-list .nav__item a {
   border: 3px solid transparent;
   transition: all 0.5s linear;
 }
 
+.nav-list.nav--open .nav__item a {
```
Comment (sja-thedude) [inline style.css:97]: - [ ] Kindly make sure to replace the correct background colour for the menu on line 81 with #6070ff.
- [ ] Kindly make sure that when the menu is displayed, the outside background of the page is blurred as per the design.
- [ ] Kindly follow the spacing between mobile menu options as shown in the video.

https://user-images.githubusercontent.com/70269507/173540660-56a34eb9-d300-49ca-93fa-38af90d23067.mov

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @bahoz-ali, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Professional readme
✔️ Good PR Description and Title
✔️ Great job so far

## Required 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (bahoz-ali) [reply style.css:97]: **thanks, I fixed all your requirements.**

**Review by KayLemba — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello  @bahoz-ali      , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

HIGHLIGHTS:

- Your project is professional 
- Your repo is professional 
- Well structured files 


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

[OPTIONAL] Kindly add a screenshot to your README file showing your webpage user Interface. You can use extentions like fireshot to capturer the entire page.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**